http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/PersistentDiskComponent.scala

```diff
@@ -108,6 +108,12 @@ object persistentDiskQuery extends TableQuery(new PersistentDiskTable(_)) {
       .map(d => (d.status, d.destroyedDate, d.dateAccessed))
       .update((DiskStatus.Deleted, destroyedDate, destroyedDate))
 
+  def updateGoogleId(id: DiskId, googleId: GoogleId, dateAccessed: Instant) =
```

Comment:
just curious, when does `googleId` change?

Reply:
We create the Disk in our db, then send the api call to google which creates the `googleId` so we have to update the `googleId` in our db after we successfully create the disk in google